build/buildRunTimeImageBases.sh

```diff
@@ -17,7 +17,7 @@ declare -r NODE_BUSTER_VERSION_ARRAY=($NODE16_VERSION $NODE14_VERSION)
 
 runtimeImagesSourceDir="$RUNTIME_IMAGES_SRC_DIR"
 runtimeSubDir=""
-runtimeImageDebianFlavor="buster"
+runtimeImageDebianFlavor=""
```

Comment:
Could you please provide more context about this change? Is there any case where this can be unset?

Reply:
It was for testing locally, adding it back.

Reply:
Actually, looks like the other one was fixed but not this one.

Reply:
updated